app/main/util/filter_sort_builder.py

```diff
@@ -41,6 +41,29 @@ def build_sorting_orders(args):
     return sorting_orders
 
 
+def build_browse_consignment_filters(args):
```

Comment:
there is an existing `_build_consignment_filters` which is called inside `_build_consignment_view_query` which is called in `browse_data`. Can we consolidate these / delete what is redundant?

Reply:
I think there was some confusion, there is still 2 lots of filtering for the conginment browse